src/coreclr/vm/frames.cpp

```diff
@@ -612,7 +612,7 @@ MethodDesc* StubDispatchFrame::GetFunction()
     {
         if (m_pRepresentativeMT != NULL)
         {
-            pMD = m_pRepresentativeMT->GetMethodDescForSlot(m_representativeSlot);
+            pMD = m_pRepresentativeMT->GetMethodDescForSlot_NoThrow(m_representativeSlot);
```

Comment:
Is `NULL` okay here?

Reply:
No, but that's ok since this function doesn't have that behavior. Its just a different algorithm for producing the same result.

Reply:
My point is if this is not supposed to return a `NULL` and we're calling the "NoThrow" then at the very least we need an assert to convey that assumption.